src/components/richLink.tsx

```diff
@@ -143,7 +143,7 @@ const RichLink = (props: {
 	const webUrl = 'https://www.theguardian.com';
 
 	const articleId = url.includes(webUrl)
-		? { 'data-article-id': url.replace(webUrl, '') }
+		? { 'data-article-id': url.replace(webUrl, '/rendered-items') }
```

Comment:
Will this work locally?

Reply:
Seems to work with https://github.com/guardian/apps-rendering/pull/943/files#diff-0ec854d4226b5921f1520f7bfae4e2eb27cdedadf904662fade4342f6a57472cR258